src/main/java/com/coveros/selenified/utilities/Reporter.java

```diff
@@ -901,6 +880,165 @@ public static String formatKeyPair(Map<String, Object> keyPairs) {
         return stringBuilder.toString();
     }
 
+    /**
+     * Formats the request parameters to be 'prettily' printed out in HTML
+     *
+     * @param params - the parameters to be formatted. Either a JSON object, or a
+     *               hashmap
+     * @return String: a 'prettily' formatted string that is HTML safe to output
+     */
+    public static String getRequestPayloadOutput(Request params, File file) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 16 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS3776)